GitUI/CommandsDialogs/FormFileHistory.cs

```diff
@@ -467,21 +471,43 @@ private void UpdateFollowHistoryMenuItems()
             followFileHistoryRenamesToolStripMenuItem.Checked = AppSettings.FollowRenamesInFileHistoryExactOnly;
         }
 
-        private void fullHistoryToolStripMenuItem_Click(object sender, EventArgs e)
+        private void showFullHistoryToolStripMenuItem_Click(object sender, EventArgs e)
         {
             ToggleFullHistoryFlag();
         }
 
-        private void ShowFullHistory_Click(object sender, EventArgs e)
+        private void fullHistoryToolStripMenuItem_Click(object sender, EventArgs e)
         {
             ToggleFullHistoryFlag();
         }
 
+        private void simplifyMergesToolStripMenuItem1_Click(object sender, EventArgs e)
```

Comment:
Why is `simplifyMergesToolStripMenuItem1`?

Reply:
Missed rename, fixed.

About UI: from now 'Simplify Merges' button is disabled if 'Show Full History' is not checked